doc/specs/stdlib_linalg.md

```diff
@@ -1382,3 +1382,112 @@ If `err` is not present, exceptions trigger an `error stop`.
 {!example/linalg/example_inverse_function.f90!}
 ```
 
+## `norm` - Computes the vector norm of a generic-rank array.
+
+### Status
+
+Experimental
+
+### Description
+
+This function computes one of several vector norms of `real` or `complex` array \( A \), depending on 
+the value of the `order` input argument. \( A \) may be an array of any rank. 
+
+Result `x` returns a `real` scalar norm value for the whole array; if `dim` is specified, `x` is an array of rank n-1, 
```

Comment:
This paragraph is mentioned just below in the "return value" section. Maybe it can be dropped from the Description section.

Reply:
[91882fa](https://github.com/fortran-lang/stdlib/pull/871/commits/91882fa9bf68871f2259033f01a26fff1389b8e4)